act/workers/argus_case.py

```diff
@@ -21,6 +21,8 @@
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+WORKER_NAME = "argus-case"
```

Comment:
we do not have a `worker_name()` in `worker` (see ussage on line 104), so we set this explicit